README.rst

```diff
@@ -107,18 +96,18 @@ Limitations
 Transaction management isn't supported
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-django-spanner always works in Django's default transaction behavior,
-``autocommit`` mode. Transactions cannot be controlled manually with
+``python-spanner-django`` always works in Django's default transaction
```

Comment:
[nit] Maybe `The ''python-spanner-django''...`?

Reply:
Also, the usage of comma after `behavior` is unclear. Maybe `...transaction behavior, the ''autocommit'' mode.` Then it's not Django's but Spanner's default transaction mode, WDYT?

Reply:
I agree that's not clear, changed it to say:

> `python-spanner-django` always works in `autocommit` mode, which is Django's default behavior even for backends that support manual transaction management.

I think this is the same sentiment but more explicit.

Reply:
> [nit] Maybe The ''python-spanner-django''...?

I don't think the package name needs an article.